.github/labels.yml

```diff
@@ -15,6 +15,11 @@
   color: '3E5E06'
   description: 'add to a merged PR to revert it (skips CI)'
 
+# Unito - notion.so/52d10dbfad474328850319e48b057a5b
+- name: "Sync: Jira"
```

Comment:
Is it truly a sync, or does it just copy?

Reply:
[RTFM](https://www.notion.so/GitHub-Jira-Integration-52d10dbfad474328850319e48b057a5b) ;-)

tl;dr It's a sync.